homeassistant/components/google_generative_ai_conversation/conversation.py

```diff
@@ -185,9 +185,6 @@ def supported_languages(self) -> list[str] | Literal["*"]:
     async def async_added_to_hass(self) -> None:
         """When entity is added to Home Assistant."""
         await super().async_added_to_hass()
-        assist_pipeline.async_migrate_engine(
```

Comment:
The conversation agent ones are newer and not sure we drop them yet

Reply:
Should we wait longer? May be add some comments?

Reply:
We should look at what release they came in, and then do +6 months or a year. There really is not much lost to keep it around.

Reply:
I'll update/add the comment's based on the introduction , so we can track more easily and keep a year of time.